components/tikv_alloc/src/jemalloc.rs

```diff
@@ -5,25 +5,86 @@ use crate::AllocStats;
 use jemalloc_ctl::{stats, Epoch as JeEpoch};
 use jemallocator::ffi::malloc_stats_print;
 use libc::{self, c_char, c_void};
-use std::{io, ptr, slice};
+use std::collections::HashMap;
+use std::{io, ptr, slice, sync::Mutex, thread};
 
 pub type Allocator = jemallocator::Jemalloc;
 pub const fn allocator() -> Allocator {
     jemallocator::Jemalloc
 }
 
+lazy_static! {
+    static ref THREAD_MEMORY_MAP: Mutex<HashMap<String, MemoryStatsAccessor>> =
+        Mutex::new(HashMap::new());
+}
+
+struct MemoryStatsAccessor {
+    allocated: jemalloc_ctl::thread::AllocatedP,
+    deallocated: jemalloc_ctl::thread::DeallocatedP,
+    num_arenas: jemalloc_ctl::arenas::NArenas,
+}
+
+pub fn add_thread_memory_accessor() {
+    let mut thread_memory_map = THREAD_MEMORY_MAP.lock().unwrap();
+    thread_memory_map.insert(
+        thread::current().name().unwrap().to_string(),
+        MemoryStatsAccessor {
+            allocated: jemalloc_ctl::thread::AllocatedP::new().unwrap(),
+            deallocated: jemalloc_ctl::thread::DeallocatedP::new().unwrap(),
+            num_arenas: jemalloc_ctl::arenas::NArenas::new().unwrap(),
```

Comment:
I think the original issue is about to query the arena ID instead of the count.

Reply:
Sg, will change.